op-e2e/actions/helpers/l2_batcher.go

```diff
@@ -136,13 +136,27 @@ func (s *L2Batcher) SubmittingData() bool {
 	return s.l2Submitting
 }
 
+// Reset the batcher state, clearing any buffered data.
+func (s *L2Batcher) Reset() {
+	s.L2ChannelOut = nil
+	s.l2Submitting = false
+	s.L2BufferedBlock = eth.L2BlockRef{}
+	s.l2SubmittedBlock = eth.L2BlockRef{}
+}
+
 // ActL2BatchBuffer adds the next L2 block to the batch buffer.
 // If the buffer is being submitted, the buffer is wiped.
 func (s *L2Batcher) ActL2BatchBuffer(t Testing) {
 	require.NoError(t, s.Buffer(t), "failed to add block to channel")
 }
 
 func (s *L2Batcher) Buffer(t Testing) error {
+	return s.BufferWithOpts(t)
+}
+
+type BlockModifier = func(block *types.Block)
+
+func (s *L2Batcher) BufferWithOpts(t Testing, opts ...BlockModifier) error {
```

Comment:
Given `opts` is optional, shouldn't we be able to just make this `Buffer` and not need the two separate functions?  All the existing calls should still work without providing any options.